server/rpc.go

```diff
@@ -171,12 +172,20 @@ func GetOrchestratorInfo(ctx context.Context, bcast Broadcaster, orchestratorSer
 	return r, nil
 }
 
+// TestDialer used for connecting to mocked server in unit tests
+var TestDialer func(string, time.Duration) (gnet.Conn, error)
+
 func startOrchestratorClient(uri *url.URL) (net.OrchestratorClient, *grpc.ClientConn, error) {
 	glog.Infof("Connecting RPC to %v", uri)
-	conn, err := grpc.Dial(uri.Host,
-		grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)),
-		grpc.WithBlock(),
-		grpc.WithTimeout(GRPCConnectTimeout))
+	opts := []grpc.DialOption{grpc.WithBlock(), grpc.WithTimeout(GRPCConnectTimeout)}
+	if TestDialer != nil {
```

Comment:
It seems confusing to leave this in without a corresponding usage example. Is there another `rpc.go` function that could use improved test coverage? It shouldn't block this PR though.

Reply:
I actually just forgot to remove it. Will remove it - it's not needed right now.